github/ContentFile.py

```diff
@@ -58,7 +56,7 @@ def decoded_content(self):
         """
         :type: bytes
         """
-        assert self.encoding == "base64", "unsupported encoding: %s" % self.encoding
```

Comment:
need to fix this

Reply:
Our assert messages are a mess anyway, but I'll put it back.